cas/incident-report.md

```diff
@@ -1,169 +1,641 @@
+- The time at which the CA Owner provided a preliminary report on its findings to the entity who filed the Certificate Problem Report (if applicable)
+- The time at which the CA Owner provided a preliminary report on its findings to the affected Subscriber(s) (if applicable)
+- The time at which the CA Owner concluded and disclosed the scope and impact of the incident
+- The time at which the CA Owner updated the scope and impact of the incident (if applicable)
+- The time(s) at which the CA Owner is expected to complete revocation of affected certificates (if applicable, but required when Whiteboard field contains ‘revocation-delay’)
+- The time(s) at which the CA Owner actually completed revocation of affected certificates (if applicable, but required when Whiteboard field contains ‘revocation-delay’)
+- The time at which the incident ended
+- The times at which issuance ceased and resumed (if applicable)
+
+**Related Incidents:** The Related Incidents section includes a list of all incidents disclosed to the ['CA Certificate Compliance' Bugzilla Component](https://bugzilla.mozilla.org/buglist.cgi?product=CA%20Program&component=CA%20Certificate%20Compliance&resolution=---) related to this incident that have taken place minimally in the last two (2) years. "Related Incidents" MUST consider incidents beyond those corresponding to the CA Owner subject of this report.
+
+| Field                           | Description                              |  
+|---------------------------------|------------------------------------------| 
+| **Related Bug ID** | The Bugzilla ID corresponding to the related incident. |
+| **Related Bug URL** | The Bugzilla URL corresponding to the related incident. |
+| **Opened date** | The date the related incident was opened. |
+| **Description** | A description of how the related incident is similar to the subject incident report. |
+
+**Root Causes:** The Root Causes section contains a detailed analysis of the conditions which combined to give rise to the issue. It is unusual for an incident to have a single root cause; often there is a confluence of several issues such as a software bug, insufficient checks, and a malformed request. Make sure that all contributing causes are identified and described, including noting when they first arose and how they avoided detection until they were discovered or identified.  
+
+| Field                           | Description                              |  
+|---------------------------------|------------------------------------------| 
+| **Description** | A detailed description of the specific issue. |
+| **Issue onset** | Date when the issue began. |
+| **Issue detection** | Date when the issue was detected or otherwise made known. |
+| **Issue resolution** | Actual or planned date and time when the issue will be considered resolved. |
+| **Symptoms that led to detection** | A detailed description of the circumstances that led to the detection of the issue. |
```

Comment:
I think that these five bullets will harm root cause analysis more than help it.

The single most common root cause across incident reports is "we did not have processes or automation in place to prevent a human from making this mistake". Whether the mistake was not renewing a CRL in time, or not noticing that a validation method didn't comply with a subtle requirement, or typoing a country code, the overall shape is the same.

And these bullets simply don't apply to most root causes with that shape. Having a text field for the country code instead of a dropdown selector isn't an issue with a meaningfully useful onset date -- it's been that way forever. The detection date for realizing you don't have an alert that fires when your root CRL is too old is "while writing this incident report"; the symptoms are "we had this incident". The resolution date is redundant with the Action Items.

The existence of these bullets -- _especially_ combined with the very minimal description for the "Description" field -- will cause incident report authors to believe they have discharged their analysis obligations when they have a pithy one-sentence answer for each bullet. I think that will lead to meaningfully worse root cause analysis than the multi-paragraph prose we see today.